src/WebsockettestHelpers.js

```diff
@@ -105,3 +105,21 @@ Math.median = function(values) {
 };
 
 
+let Logger = (function () {
```

Comment:
Is there some mechanism to disable the logger or set it to silent? (since this can impact runtime performance)
If so, how can the error level be set globally?